src/react-renderer/src/SquareTransform.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from "react";
+import { svgTransformString, toHex } from "./Util";
+import draggable from "./Draggable";
+import { IGPIPropsDraggable } from "./types";
+
+class SquareTransform extends React.Component<IGPIPropsDraggable> {
+  public render() {
+    const { shape } = this.props;
+    const { canvasSize } = this.props;
+    const { onClick } = this.props;
+    // const [x, y] = toScreen([shape.x.contents, shape.y.contents], canvasSize);
+    const color = toHex(shape.color.contents);
+    const alpha = shape.color.contents[3];
+    const strokeColor = toHex(shape.strokeColor.contents);
+    const side = 1.0;// shape.side.contents;
+    const strokeWidth = shape.strokeWidth.contents;
+    const transformStr = svgTransformString(shape.transformation.contents, canvasSize);
+
+    return (
+      <rect
+        x={-0.5}
+        y={-0.5}
+        width={side}
```

Comment:
It should start in the frontend with unit width and height, and should be scaled to the right size by the transform in `transformStr`. See the approach taken for RectangleTransform: https://github.com/penrose/penrose/blob/transforms/src/react-renderer/src/RectangleTransform.tsx#L26